### PR TITLE
gaia-flex: add orkun validator as seed

### PR DESCRIPTION
**Pull request by orkunkl**

### CODE REVIEW

File: gaia-flex/config/config.toml [hunk 1]
```diff
@@ -0,0 +1,335 @@
+# This is a TOML config file.
```
Comment (ethanfrey) [inline gaia-flex/config/config.toml]: Maybe we don't need to check this in?

File: gaia-flex/config/config.toml [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+# If empty, will use the same port as the laddr,
+# and will introspect on the listener or use UPnP
+# to figure out the address.
+external_address = ""
+
+# Comma separated list of seed nodes to connect to
+seeds = ""
+
+# Comma separated list of nodes to keep persistent connections to
+persistent_peers = "5577df9fc114ead3cbc2b9a3bfab670510004f39@168.119.56.205:26656"
```
Comment (ethanfrey) [inline gaia-flex/config/config.toml]: Ah, this is the line you are adding.

I would just update this line: https://github.com/CosmWasm/testnets/blob/master/gaia-flex/defaults.env#L16

(and remove the non-existent faucet)

**Review by ethanfrey — COMMENTED** [review]
Maybe better to do the same as coralnet and update the defaults.env?

File: gaia-flex/config/config.toml (hunk 2 above)
Comment (orkunkl) [reply gaia-flex/config/config.toml]: You are right. I forgot that variables exist.

**Review by ethanfrey — APPROVED** [review]
lgtm

**Pull request merged**